201-ultra-managed-disk/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
remove this and use defaultValues in the template file

Reply:
Addressed some of the variables. Please let me know if there are others you are concerned about.